pyproject.toml

```
@@ -3,7 +3,8 @@ requires = [
     "setuptools; python_version != '3.3'",
     "setuptools<40.0; python_version == '3.3'",
     "wheel",
-    "setuptools_scm<8.0"
+    "setuptools_scm; python_version >= '3.0'",
+    "setuptools_scm<8.0; python_version < '3.0'"
```

```suggestion
    "setuptools_scm; python_version >= '3.6'",
    "setuptools_scm<8.0; python_version < '3.6'"
```

https://github.com/dateutil/dateutil/issues/1348

---

I see what @pganssle said in https://github.com/dateutil/dateutil/pull/1349#issuecomment-1974059666. He's correct that this won't work:

* If you build on Python >= 3.6, you get a wheel built with the latest `setuptools_scm`
* If you build on Python < 3.6, you get a wheel built with `setuptools_scm` < 8.0

You can only pick one, and in either situation you'll have unhappy users :)